src/style.css

```diff
@@ -0,0 +1,152 @@
+:root {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+/* BLOCK "CARD" */
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
it is better to define it in * selector

Reply:
I don't agree. If I'll define this properties in * selector - when I'll need to multiply or put this block "card" into some code, it will make me troubles

Reply:
I suggest you'll have more troubles if you avoid applying `box-sizing: border-box` for all elements. Because this behavior we would like to see for our box almost for all cases. 
So, this is really useful recommendation that save your time in a future
However, you can decide by yourself

Reply:
Thanks, I'll definitely use it in the future.
But for this task I'll leave it as it is :)